<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ecdadı Tanı v1.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lbl_bg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>600</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>                                                                                                 bg</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_zaman_skor">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>21</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="zaman">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>69</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">style{
rgb(255, 255, 255)
}


</string>
    </property>
    <property name="text">
     <string>Zaman    :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_yakala">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>69</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Yakaldın :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_kacan">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>69</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Kaçırdın :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_yakala_skor">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>21</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_kacan_skor">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>21</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
